tests/transactions/builder/test_htlc_claim.py

```diff
@@ -0,0 +1,27 @@
+from crypto.constants import TRANSACTION_HTLC_CLAIM, TRANSACTION_TYPE_GROUP
+from crypto.transactions.builder.htlc_claim import HtlcClaim
+
+
+def test_htlc_claim_transaction():
+    """Test if timelock transaction gets built
+    """
+    lock_transaction_id = '943c220691e711c39c79d437ce185748a0018940e1a4144293af9d05627d2eb4'
+
+    # @TODO: Not sure here, shouldn't the builder take a bytes encoded string instead of accepting a plain string ?
```

Comment:
https://github.com/ArkEcosystem/AIPs/blob/master/AIPS/aip-102.md#2htlc_claim the secret should be a hash, not a plain string. The user will have to provide a hash of their secret, which they pass to the builder instead of the plain secret itself